README.md

```diff
@@ -6,41 +6,184 @@
 
 ![](src/main/resources/META-INF/pluginIcon.svg)
 
+Git Machete plugin is a robust tool that simplify your git related workflow.
+It's a great completion of IntelliJ version control system.
+You can with easy rebase branches, do pull, push or reset.
+Just define what you would like your branch tree to look like and leave the rest to Git Machete.
+It gives you simple view of what's going on in your repository.
+Just look at the edge colors or status descriptions to see what should be rebased, pushed, pulled or reset.
+
+With this plugin you can simply maintain small pull request that are easy to review.
+
 This is a port of [git-machete](https://github.com/VirtusLab/git-machete#git-machete) into an IntelliJ plugin.
 
 This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete).
 
+**It is compatible with all JetBrains products except Android Studio.
+Minimum required version is 2020.1**.
+
 ![](docs/sample-workflow.gif)
 
 
-## Logging
+## Installing
+
+### Install from JetBrains Marketplace
+
+This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete). <br/>
+To install this plugin go to `File` -> `Settings` -> `Plugins`, then make sure you are on `Marketplace` tab (not `Installed`), in search box type `Git Machete` and click `Install`.
+Plugin will be installed. After installation depending on IDE version restart may be required. In that case just click `Restart IDE` and confirm that action in a messagebox.
+
+
+## How it works
+
+Git Machete IntelliJ Plugin is a port of a great console tool - [git-machete](https://github.com/VirtusLab/git-machete#git-machete) - into an IntelliJ plugin.
+Git Machete can help you manage your repository branches and keep it in sync with each other and its counterparts on a remote repository.
+Let's see how this plugin can help you:
+
+### Where to find this plugin window
+
+Git Machete IntelliJ Plugin is available under `Git` section (in the bottom of the IDE) in the `Git Machete` tab.
```

Comment:
IMO `Tool Window` it too technical for average IntelliJ user. I'd name this `tab` (this is probably most clear) and `Git Machete tab` rename to `Git Machete subtab`

Reply:
Are we sure that we want to name our tab `Git Machete subtab` in the one place and `(Open) Git Machete Tab` in the other? This may raise doubts. :confused: 

I would rather stay with the `section - tab` naming (instead of `tab - subtab`).
As we have no data (stats, anything) about an average user let's provide the "technical" name `Tool Window" within the parentheses too. WDYT?

Reply:
Ok, so finally I'd stay with `tool window` - `tab` naming